### PR TITLE
Update reset_xvf3510.py for 3610 support

### PR DESCRIPTION
**Pull request by ed-xmos**
3610 pithat needs level shifters turned on via I2C io expander so I2S can reach 1v8 IO on the chip

### CODE REVIEW

**Comment by DaveAtkinsonXMOS** [comment]
Ed, as requested of me, I've made some change to the vocalfusion rpi repo to support an argument of 'xvf3610' (and made corresponding changes to vocalfusion-avs-setup). I have put the changes on github.com/xmos/vocalfusion-rpi-setup branch feature/xvf3610_support_patch and github.com/xmos/vocalfusion-avs-setup branch feature/xvf3610_support respectively). Perhaps you could test them and merge the patch branch into this?

File: setup.sh [hunk 1]
```diff
@@ -51,8 +51,11 @@ sudo sed -i -e 's/dtparam=i2c_arm=on$/dtparam=i2c_arm=on\ndtparam=i2c_arm_baudra
 sudo raspi-config nonint do_spi 1
 sudo raspi-config nonint do_spi 0
 
-echo "Installing Raspberry Pi kernel headers"
-sudo apt-get install -y raspberrypi-kernel-headers
+KERNEL_HEADERS_PACKAGE=raspberrypi-kernel-headers
+if ! dpkg -s $KERNEL_HEADERS_PACKAGE &> /dev/null; then
```
Comment (lucianomartin) [inline setup.sh:55]: Can you add a comment about why we do this check?

**Review by lucianomartin — CHANGES_REQUESTED** [review]
Just one comment.
The changes look good and they are working as expected!

File: setup.sh (hunk 1 above)
Comment (DaveAtkinsonXMOS) [reply setup.sh:55]: Of course...

**Review by lucianomartin — APPROVED** [review]
Looks good!

**Pull request merged**